platform/jobs/edxPlatformAccessibilityPr.groovy

```diff
@@ -107,11 +113,14 @@ secretMap.each { jobConfigs ->
                 orgWhitelist(ghprbMap['orgWhiteList'])
                 extensions {
                     commitStatus {
-                        context('jenkins/a11y')
+                        context(jobConfig['context'])
                     }
                 }
             }
         }
+
+        configure GHPRB_WHITELIST_BRANCH(jobConfig['whitelistBranchRegex'])
```

Comment:
Is this a `configure` statement b/c this particular function is not available in the ghprb DSL?